superset-frontend/src/views/CRUD/data/database/DatabaseModal/ExtraOptions.tsx

```diff
@@ -77,6 +77,7 @@ const ExtraOptions = ({
             />
             <InfoTooltip
               tooltip={t('Allow this database to be queried in SQL Lab')}
+              viewBox="0 -2 24 24"
```

Comment:
Can we make this a constant since all the Tooltip.viewBox have the same value?

Reply:
You got it! Fixed in [`this commit`](https://github.com/apache/superset/pull/15234/commits/d9ec9e69a9c6dad602cd52329b22d3e25ac1890e).